popover-api/popover-hint/index.html

```diff
@@ -31,32 +31,48 @@
           Menu C
         </button>
       </section>
-    </div>
-
-    <p class="help-para">
-      Press the buttons to show the auto popovers. Hover or focus the buttons to
-      show the hint popovers. When an auto popover is shown, you can show the
-      hint popovers without hiding it. See
-      <a
-        href="https://developer.mozilla.org/en-US/docs/Web/API/Popover_API/Using#using_hint_popover_state"
-        >Using "hint" popover state</a
-      >
-      for more information. Note that in non-supporting browsers, the position
-      of the popovers is broken.
-    </p>
+      <div id="submenu-1" popover="auto">
+        <button>Option A</button><br /><button>Option B</button>
+      </div>
+      <div id="submenu-2" popover="auto">
+        <button>Option A</button><br /><button>Option B</button>
+      </div>
+      <div id="submenu-3" popover="auto">
+        <button>Option A</button><br /><button>Option B</button>
+      </div>
 
-    <div id="submenu-1" popover="auto">
-      <button>Option A</button><br /><button>Option B</button>
-    </div>
-    <div id="submenu-2" popover="auto">
-      <button>Option A</button><br /><button>Option B</button>
-    </div>
-    <div id="submenu-3" popover="auto">
-      <button>Option A</button><br /><button>Option B</button>
+      <div id="tooltip-1" class="tooltip" popover="hint">Tooltip A</div>
+      <div id="tooltip-2" class="tooltip" popover="hint">Tooltip B</div>
+      <div id="tooltip-3" class="tooltip" popover="hint">Tooltip C</div>
```

Comment:
I would suggest putting tooltips right next to the buttons they’re triggered by. First of all, this is something you would most likely do in a real-world component to contain everything. Second, it might help to position them next to the buttons as a fallback for browsers that don’t support the `hint` value yet:

```html
<button
  popovertarget="submenu-1"
  popovertargetaction="toggle"
  id="menu-1">
  Menu A
</button>

<div id="tooltip-1" class="tooltip" popover="hint">
  Tooltip A
</div>
```